app/javascript/app/platform-authenticator.js

```diff
@@ -0,0 +1,13 @@
+function platformAuthenticator() {
+  const isEnabled = document.querySelector('[data-platform-authenticator-enabled]');
```

Comment:
Why is this behind a feature flag?

Reply:
I discussed this with @konklone and was concerned that there might be a popup in some browser implementation which would make for a lousy experience since this could be called from any authenticated endpoint.  We decided to add a feature flag to be safe.